drivers/go/device.ts

```diff
@@ -36,7 +37,13 @@ export class GoCharger extends Homey.Device {
     this.cronTasks.push(
       cron.schedule('0,30 * * * * *', () => this.pollValues()),
```

Comment:
Should there be a jitter on pollVales too? Run every whole and half minute. Same for all users I think.

Reply:
Hi, jitter is helpful on every job that is running on the schedule, but it makes more difference on any request that potentially has big response body (just due to the memory constraints on the backend when you have too many in-flight requests).

We will have a look there after we inspect the workload from Homey more closely.

Thanks for the good work and merry Christmas!